ui/js/directives/treeherder/resultsets.js

```diff
@@ -39,17 +39,12 @@ treeherder.directive('thAuthor', function () {
     return {
         restrict: "E",
         link: function (scope, element, attrs) {
-            var userTokens = attrs.author.split(/[<>]+/);
-            var email = "";
-            if (userTokens.length > 1) {
-                email = userTokens[1];
-            }
-            scope.authorName = userTokens[0].trim();
-            scope.authorEmail = email;
+            var auth = attrs.author;
+            scope.authorEmail = /\<(.*?)\>+/.test(auth) ? auth.match(/\<(.*?)\>+/)[1] : auth;
```

Comment:
I think it would be best to not repeat the regex pattern. I'd first assign the result of the `.match()` to a variable, then use that in the ternary like `foo ? foo[0] : attrs.author` :-) (At that point `attrs.author` will only be used twice, so likely not worth assigning to it's own variable then.)